body.tex

```diff
@@ -692,7 +699,95 @@ \subsection{Reporting Cycle}
+
+\subsection{Risk-aware budgeting and Requests Beyond Target}
+
+The ground-up spending plan developed during the annual cycle is to cover operations work defined by the milestone activities, as well as any regular work captured in bucket epics.
+This work includes mitigation of risks held by the teams, as described in the response plans in the risk register.
+Any risks that are realized during the year will require action that is not budgeted for.
+Additional spending to address realized risks is enabled by Rubin's request beyond target (RBT) process, whereby reserve funds are released for use by the departments in response to a specific mid-year request made in a custom Jira ticket.
+
+Standard terminology for financial reserves is that ``management reserve'' describes funds held back to pay for the unknown unknowns, while ``contingency reserve'' is used for known unknowns (i.e.\ the risks in the risk register).
+Rubin Operations' two funding agencies offer differing guidelines on how reserve should be managed.
+DOE allows reserves to be built up and carried forward at SLAC, and used as both contingency and management reserve.
+DOE HEP holds additional reserves centrally, which can also be used as both contingency and management reserve.
+NSF AST allows line item budget over-estimation by up to 10%, to allow for increases in cost.
+These generate carry-forward (along with any other under-spending that may occur), which we use as contingency reserve.
+NSF also holds reserves of both kinds centrally, to cover high cost impact realized risks as well as unexpected events.
+
+The cost impact analysis of the Rubin Operations risk register informs the level of contingency reserve needed.
+Residual (post-mitigation) cost exposure for each risk is computed by taking the  product of the residual probability of the risk being realized (in any given year) and the likely annual cost of addressing that risk.
+The total cost exposure is then the sum, over all risks in the register, of these products.
+
+Currently, the total cost exposure, and hence the needed level of contingency reserve, is approximately 15\% of the total annual operating budget at each operations partner.
+Included in the risk register is the risk of federal funding allocations being reduced, with a likely cost (if realized in any given year) of some \$1.5M per operations partner.
+(At SLAC, for example, this would cover about 7\% of the operating expense -- enough to pay the labor bills during a 1 month funding allocation delay.)
+
+Reserve maintenance is accounted for in our federal budget requests, either as an explicit difference between cost and budget on the DOE side, or as line by line allowances for increases in cost on the NSF side.
+At both operations partners the reserve is maintained as carry-forward from one year's spending to the next, with it being spent down to zero either during the post-operations period (at SLAC), or at the end of each 5-year renewal period at NOIRLab.
```

Comment:
I would revise as "At both operations partners carry forward is maintained from one year's plan to the next, with it being spent down to zero either during the post-operations period (at SLAC), or before the end of each 5-year renewal period at NOIRLab. Each year a plan is described in the POP for how carry forward is allocated."

Reply:
OK, thanks!